src/app/view/perspectives/gantt-chart/util/gantt-chart-util.ts

```diff
@@ -192,7 +192,8 @@ export function isGanttConfigChanged(viewConfig: GanttChartConfig, currentConfig
     return true;
   }
 
-  return ganttConfigCollectionsChanged(viewConfig.collections || {}, currentConfig.collections || {});
+  const isChanged = ganttConfigCollectionsChanged(viewConfig.collections || {}, currentConfig.collections || {});
+  return isChanged;
```

Comment:
I think you can revert these changes back.

Reply:
Correct, this was only for debugging purposes!